SU-HSDO/stanford_mrc
# MRCD8-222 past events component

# READY FOR REVIEW

# Summary
- Past events pattern; title and short description

# Needed By (Date)
- End of sprint (2/15)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Ensure that the past events pattern is showing on the /patterns page and has the title and description displaying only

# Affected Projects or Products
- stanford_mrc
- MRC

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/MRCD8-222

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
